requirements.txt

```diff
@@ -0,0 +1,36 @@
+alembic==0.8.9
+aniso8601==1.2.0
+click==6.6
```

Comment:
Ensure that you only include packages you are using explicitly, you can remove packages that are installed as dependencies of other packages.